src/personen/personen.lambda.ts

```diff
@@ -80,13 +98,23 @@ export async function callHaalCentraal(content: string, personenSecrets: Persone
       },
     );
 
-    const data = await resp.json();
-
-    return {
-      statusCode: resp.status,
-      body: JSON.stringify(data),
-      headers: { 'Content-Type': 'application/json' },
-    };
+    var data: any;
```

Comment:
var wordt afgeraden om een aantal redenen. Aub const en eventueel let gebruiken.
Ik zie hem hierboven ook staan, dat is eerder dan niet opgevallen.

Reply:
verwijderd